src/ajax/parseXML.js

```diff
@@ -9,7 +9,8 @@ jQuery.parseXML = function( data ) {
 		return null;
 	}
 
-	// Support: IE9
+	// Support: IE 9-11 only
```

Comment:
I can see the benefit of either approach and I'm not sure it matters. It's sometimes handy to know whether older browsers exhibited this as well, and our official browser support is mentioned elsewhere. In a practical lazy-me sense, though, I wouldn't want to go back and edit Support comments when we drop support for older browsers, seems kind of tedious.